deploy/manifests/dev/us-east-2/tenant/storetheindex/assigner/config.json

```diff
@@ -0,0 +1,64 @@
+{
+  "Version": 1,
+  "Identity": {
+    "PeerID": "",
+    "PrivKey": ""
+  },
+  "Assignment": {
+    "FilterIPs": false,
+    "IndexerPool": [
+      {
+        "AdminURL": "http://ber.dev.cid.contact:3002"
+        "IngestURL" "http://ber.dev.cid.contact:3001"
```

Comment:
This file is not a valid `json`. `:` is missing on this line, `,` is missing at the end of `AdminURL` value. Ditto re `cali` config.